cmake/GhcHelper.cmake

```diff
@@ -22,6 +22,9 @@ if ("${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang" AND (CMAKE_CXX_COMPILER_VERSION V
     if(${CMAKE_SYSTEM_NAME} MATCHES "(SunOS|Solaris)")
         target_link_libraries(filesystem_test xnet)
     endif()
+    if(${CMAKE_SYSTEM_NAME} MATCHES "(Haiku)")
+        target_link_libraries(filesystem_test -lnetwork)
```

Comment:
the `-l` prefix can be removed, just the library name (i.e. `network`) is enough.

Reply:
Thanks on the comments, will check them out 👍